src/connectors/breadcrumb/connectBreadcrumb.js

```diff
@@ -122,7 +140,7 @@ export default function connectBreadcrumb(renderFn, unmountFn) {
           );
         };
 
-        this._refine = function(facetValue) {
+        renderingRefine = function(facetValue) {
```

Comment:
can be an arrow function.
I think it's best to use them when possible as they don't force you to think about context.